### ISSUE TITLE
Music, lobby art and other server customizations need to be moved to a different resource source

### ISSUE THREAD

**Issue by FairlySadPanda** [space-wizards/space-station-14#28929]
## Description
Lobby music and art is currently stored on this repo. This means servers pulling the game need to pull all the art and music, and Wizden music/art defaults because the assumed "default" across the growing SS14 space. But we know that both of these resources are really commonly customized by downstreams, especially downstreams who do not have to be addicted to the sweet, delcious copyright safety that we do.

Move music and large art into a server resource source that can be migrated off-repo. This would allow different server owners to easily customize their server lobby feel without needing repo edits.

Consider also migrating Wizden setup off-repo as part of this.

**Comment by lzk228** [comment]
i really dont see any issue here

**Comment by affonsoamendola** [comment]
Yeah its a great idea, I think what also is needed is a default placeholder image that tells the person that just forked this repo "Hey! Add your own lobby images/music/jukebox here:" And then point to how they can do it
